public/app/features/dashboard-scene/inspect/HelpWizard/utils.ts

```diff
@@ -62,15 +63,31 @@ export function getGithubMarkdown(panel: VizPanel, snapshot: string): string {
 }
 
 export async function getDebugDashboard(panel: VizPanel, rand: Randomize, timeRange: TimeRange) {
-  let saveModel;
+  let saveModel: ReturnType<typeof gridItemToPanel> = { type: '' };
   const isLibraryPanel = panel.parent instanceof LibraryVizPanel;
   const gridItem = (isLibraryPanel ? panel.parent.parent : panel.parent) as DashboardGridItem;
+  const scene = panel.getRoot() as DashboardScene;
 
   if (isLibraryPanel) {
     saveModel = {
       ...gridItemToPanel(gridItem),
       ...vizPanelToPanel(panel),
     };
+  } else if (scene.state.editPanel) {
+    // If panel edit mode is open when the user chooses the "get help" panel menu option
+    // we want the debug dashboard to include the panel with any changes that were made while
+    // in panel edit mode.
+    const gridItems = (scene.state.body as SceneGridLayout).state.children as DashboardGridItem[];
```

Comment:
I think we need to pull gridItems from rows as well. Otherwise the issue will still exist for panels in rows when clicking "get help" in panel edit.

Reply:
Victor is right. The get help view is empty when inspecting a panel that is within a repeated row.